test/unit/web-runtime/next-cookies.test.ts

```diff
@@ -68,28 +68,32 @@ it('reflect .delete into `set-cookie`', async () => {
   expect(Object.fromEntries(response.headers.entries())['set-cookie']).toBe(
     'foo=bar; Path=/'
   )
-  expect(response.cookies.get('foo')).toBe('foo=bar; Path=/')
+  expect(response.cookies.get('foo')).toEqual({ Path: '/', foo: 'bar' })
+  expect(response.cookies.getRaw('foo')).toBe('foo=bar; Path=/')
 
   response.cookies.set('fooz', 'barz')
   expect(Object.fromEntries(response.headers.entries())['set-cookie']).toBe(
     'foo=bar; Path=/, fooz=barz; Path=/'
   )
-  expect(response.cookies.get('fooz')).toBe('fooz=barz; Path=/')
+  expect(response.cookies.get('fooz')).toEqual({ Path: '/', fooz: 'barz' })
+  expect(response.cookies.getRaw('fooz')).toBe('fooz=barz; Path=/')
 
   const firstDelete = response.cookies.delete('foo')
   expect(firstDelete).toBe(true)
   expect(Object.fromEntries(response.headers.entries())['set-cookie']).toBe(
     'foo=; Path=/; Expires=Thu, 01 Jan 1970 00:00:00 GMT, fooz=barz; Path=/'
   )
 
-  expect(response.cookies.get('foo')).toBe(undefined)
+  expect(response.cookies.get('foo')).toEqual(undefined)
+  expect(response.cookies.getRaw('foo')).toBe(undefined)
```

Comment:
I think we should make them consistent since they're both `undefined`:

```suggestion
  expect(response.cookies.get('foo')).toBe(undefined)
  expect(response.cookies.getRaw('foo')).toBe(undefined)
```